lib/js/components/Toast/Toast.vue

```diff
@@ -1,65 +1,55 @@
 <template>
-	<Teleport :to="appendToElement" :disabled="disableTeleport">
-		<span class="ds-toast" :style="styles">
-			<span
-				ref="toastCard"
-				:class="{
-					'ds-toast__absoluteWrapper': true,
-					'-ds-size-small': size === TOAST_SIZES.SMALL,
-					'-ds-size-medium': size === TOAST_SIZES.MEDIUM,
-				}"
-			>
-				<ds-card
-					:loading-bar-color="color"
-					has-loading-bar
-					:loading-bar-time="disappearingTimeout"
-				>
-					<template #content>
-						<slot name="content" />
-					</template>
-					<template #footer>
-						<div class="ds-toast__footerButtons">
-							<ds-button
-								v-if="footerSecondaryButtonText.length"
-								:color="buttonSecondaryColor"
-								:icon-right="footerSecondaryButtonIcon"
-								:type="BUTTON_TYPES.OUTLINED"
-								:radius="BUTTON_RADIUSES.ROUNDED"
-								@click="$emit('secondary-button-click')"
-								>{{ footerSecondaryButtonText }}</ds-button
-							>
-							<ds-button
-								v-if="footerPrimaryButtonText.length"
-								:color="buttonPrimaryColor"
-								:icon-right="footerPrimaryButtonIcon"
-								:radius="BUTTON_RADIUSES.ROUNDED"
-								@click="$emit('primary-button-click')"
-								>{{ footerPrimaryButtonText }}</ds-button
-							>
-						</div>
-					</template>
-				</ds-card>
-			</span>
-		</span>
-	</Teleport>
+	<div
+		:style="styles"
+		:class="{
+			'ds-toast': true,
+			'-ds-size-small': size === TOAST_SIZES.SMALL,
+			'-ds-size-medium': size === TOAST_SIZES.MEDIUM,
+		}"
+	>
+		<ds-card :loading-bar-color="color" has-loading-bar :loading-bar-time="disappearingTimeout">
+			<template #content>
+				<slot name="content" />
+			</template>
+			<template #footer>
+				<div class="ds-toast__footerButtons">
```

Comment:
should we render empty div when footerSecondaryButtonText.length = 0 and footerPrimaryButtonText.length = 0?